doc/interfaces/etas.rst

```diff
@@ -0,0 +1,33 @@
+ETAS
+====
+
+This interface adds support for CAN interfaces by `ETAS`_.
+The ETAS BOA_ (Basic Open API) is used.
+Install the "ETAS ECU and Bus Interfaces – Distribution Package".
+Only Windows is supported.
```

Comment:
I propose to rephrase this line as below:

Only Windows is supported by this interface. The Linux kernel v5.13 (and greater) natively supports ETAS ES581.4, ES582.1 and ES584.1 USB modules. To use these under Linux, please refer to :ref:`socketcan`.